src/main/java/com/example/online_shop_api/Service/EmployeeService.java

```diff
@@ -119,8 +116,8 @@ private boolean isPhoneNumberInDB(String phoneNumber) {
         return employeeRepository.findByPhoneNumber(phoneNumber).isPresent();
     }
 
-    public ResponseEntity<MyUserDetails> showEmployeeProfile(Authentication authentication) {
-        MyUserDetails userDetails = (MyUserDetails) authentication.getPrincipal();
-        return ResponseEntity.ok(userDetails);
-    }
+//    public ResponseEntity<MyUserDetails> showEmployeeProfile(Authentication authentication) {
+//        MyUserDetails userDetails = (MyUserDetails) authentication.getPrincipal();
+//        return ResponseEntity.ok(userDetails);
+//    }
```

Comment:
also here

Reply:
This is temporarily commented out code until the security is applied.

Reply:
Also here - we will change the return type to be EntityResponseDto.